src/transforms/log_to_metric.rs

```diff
@@ -635,15 +635,15 @@ fn get_distribution_value(log: &LogEvent) -> Result<MetricValue, TransformError>
     })
 }
 
-fn get_histogram_value(log: &LogEvent) -> Result<MetricValue, TransformError> {
+fn get_histogram_value(log: &LogEvent, key: &str) -> Result<MetricValue, TransformError> {
     let event_buckets = log
-        .get(event_path!("histogram", "buckets"))
+        .get(event_path!(key, "buckets"))
         .ok_or_else(|| TransformError::PathNotFound {
-            path: "histogram.buckets".to_string(),
+            path: format!("{}.buckets", key),
```

Comment:
Nit:

```suggestion
            path: format!("{key}.buckets"),
```

This should be applied to all `format!` calls.